### DISCUSSION TITLE
noobfriendly tutorial for ldap

### DISCUSSION THREAD

**Discussion by SteffenBernd** [netbox-community/netbox-docker#981] [Getting Help]
Hey everyone,
is there a noobfriendly tutorial on ldap? the wiki is kinda confusing and i dont want to mess my netbox environment up ;)
Greetings
Steffen

**Comment by SteffenBernd** [comment]
No one? :-/

**Reply by fatman00** [reply]
Have you looked at this?
https://github.com/netbox-community/netbox-docker/wiki/LDAP
